model/model.go

```diff
@@ -36,9 +36,10 @@ const (
 type PlayerColor int8
 
 const (
-	Green PlayerColor = iota
-	Blue
-	Red
+	UnsetColor PlayerColor = 0
+	Green      PlayerColor = 1
+	Blue       PlayerColor = 2
+	Red        PlayerColor = 3
```

Comment:
Any reason you ditched `iota` and auto incrementing?

Reply:
:nods: https://github.com/joshprzybyszewski/cribbage/pull/43#discussion_r426156986